src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx

```diff
@@ -99,14 +85,37 @@ const TabularModificationForm = () => {
                 header: true,
                 skipEmptyLines: true,
                 dynamicTyping: true,
+                comments: '#',
                 complete: (results: Papa.ParseResult<object>) => {
-                    postProcessFile(results.data);
+                    clearErrors(MODIFICATIONS_TABLE);
+                    setValue(MODIFICATIONS_TABLE, results.data, {
+                        shouldDirty: true,
+                    });
+                },
+                transformHeader: (header: string) => {
+                    // transform header to modification field
+                    const transformedHeader = TABULAR_MODIFICATION_FIELDS[
+                        getValues(TYPE)
+                    ]?.find(
+                        (field) => intl.formatMessage({ id: field }) === header
+                    );
+                    return transformedHeader ?? header;
+                },
+                transform: (value: string) => {
```

Comment:
we want to do this tranformation ? It's weird to specify true|false as accepted values and then accept oui|non